src/frontend/src/tests/components/send/SendInputAmount.spec.ts

```diff
@@ -0,0 +1,47 @@
+import SendInputAmount from '$lib/components/send/SendInputAmount.svelte';
+import en from '$tests/mocks/i18n.mock';
+import { fireEvent, render } from '@testing-library/svelte';
+
+describe('SendInputAmount', () => {
+	const amount = 10.25;
+
+	const props = {
+		amount: amount,
+		calculateMax: () => amount * 2
+	};
+
+	it('should render an input', () => {
+		const { container } = render(SendInputAmount, { props });
+
+		const input: HTMLInputElement | null = container.querySelector('input');
+		expect(input?.value).toBe(props.amount.toString());
+	});
+
+	it('should not render a max button if calculateMax is not provided', () => {
+		const { calculateMax: _, ...newProps } = props;
+		const { container } = render(SendInputAmount, { props: newProps });
+
+		const button: HTMLButtonElement | null = container.querySelector('button');
```

Comment:
this test would be more future prone if we use the `data-tid` selector instead of querying for the existance of a button

ditto for other tests